### PR TITLE
Bump images to 4.14.0

### PR DESCRIPTION
**Pull request by luisfcorreia**
Bump images to 4.14.0

Removed selenium-http-jdk-client-xxx.jar from image as we're now using Java 11

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by amardeep2006** [comment]
hi @luisfcorreia  I see multiple reference to /opt/selenium/selenium-http-jdk-client.jar in NodeBase/start-selenium-node.sh . It is present in almost every Image (Hub,Router etc. etc.)
Can that cause some issue on runtime ?
Few references for example : 
https://github.com/SeleniumHQ/docker-selenium/blob/69d9349c6eefcdcaa9b899547caceb86f9f7998d/NodeBase/start-selenium-node.sh#L39C1-L39C10

https://github.com/SeleniumHQ/docker-selenium/blob/69d9349c6eefcdcaa9b899547caceb86f9f7998d/NodeBase/start-selenium-node.sh#L54

**Comment by luisfcorreia** [comment]
yes @amardeep2006, you may be right, I'll look into it

File: Base/Dockerfile [hunk 1]
```diff
@@ -73,10 +73,10 @@ COPY supervisord.conf /etc
 RUN  mkdir -p /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
   && touch /opt/selenium/config.toml \
   && chmod -R 777 /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor /etc/passwd \
-  && wget --no-verbose https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.13.0/selenium-server-4.13.0.jar \
+  && wget --no-verbose https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.14.0/selenium-server-4.14.0.jar \
     -O /opt/selenium/selenium-server.jar \
-  && wget --no-verbose https://repo1.maven.org/maven2/org/seleniumhq/selenium/selenium-http-jdk-client/4.13.0/selenium-http-jdk-client-4.13.0.jar \
-    -O /opt/selenium/selenium-http-jdk-client.jar \
+  #&& wget --no-verbose https://repo1.maven.org/maven2/org/seleniumhq/selenium/selenium-http-jdk-client/4.14.0/selenium-http-jdk-client-4.14.0.jar \
+  #  -O /opt/selenium/selenium-http-jdk-client.jar \
```
Comment (diemol) [inline Base/Dockerfile]: Please delete this lines instead of commenting on them.

File: charts/selenium-grid/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this helm chart will be documented in this file.
 
+## :heavy_check_mark: 0.23.0
+
+### Changed
+- Update image tag to 4.14.0-20231010
+
```
Comment (diemol) [inline charts/selenium-grid/CHANGELOG.md]: This should be `0.22.1`.

File: charts/selenium-grid/Chart.yaml [hunk 3]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: selenium-grid
 description: A Helm chart for creating a Selenium Grid Server in Kubernetes
 type: application
-version: 0.22.0
-appVersion: 4.13.0-20231004
+version: 0.23.0
```
Comment (diemol) [inline charts/selenium-grid/Chart.yaml]: This should be `0.22.1`.

**Review by diemol — CHANGES_REQUESTED** [review]

**Comment by luisfcorreia** [comment]
Proposed changes applied

**Comment by baflQA** [comment]
Btw, there is already a newer version of Selenium - [4.14.1]

**Comment by luisfcorreia** [comment]
Forgot there is a new Selenium Java release in the meantime (thanks Bartek)

File: charts/selenium-grid/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this helm chart will be documented in this file.
 
+## :heavy_check_mark: 0.22.1
+
+### Changed
+- Update image tag to 4.14.0-20231010
```
Comment (diemol) [inline charts/selenium-grid/CHANGELOG.md]: I was thinking about this again; this file cannot be changed before the release happens. Can you please revert this change and send it in a separate PR?

File: charts/selenium-grid/Chart.yaml [hunk 5]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: selenium-grid
 description: A Helm chart for creating a Selenium Grid Server in Kubernetes
 type: application
-version: 0.22.0
-appVersion: 4.13.0-20231004
+version: 0.22.1
```
Comment (diemol) [inline charts/selenium-grid/Chart.yaml]: I was thinking about this again; this file cannot be changed before the release happens. Can you please revert this change and send it in a separate PR?

**Review by diemol — CHANGES_REQUESTED** [review]

**Comment by luisfcorreia** [comment]
Reverting as requested

**Review by diemol — APPROVED** [review]
Let's wait for CI to pass and then let's merge.

**Pull request merged**